Simplify FROM, TO logic

cclauss opened the pull request.

Commit:
Simplify FROM, TO logic

Commit:
Update aboki

akinjide commented on aboki:
```diff
@@ -34,7 +34,6 @@ from bs4 import BeautifulSoup
 from docopt import docopt
 from requests import get, post
 
-
```
The extra whitespaces for readability

akinjide commented:
Hi @cclauss Awesome stuff. 
I have updated the documentation and added few changes at my end, would it be fine to update your PR.

akinjide commented:
@cclauss you can update your PR. ✅

Commit:
Resolve the conflicts

OK to merge review and the code but **do not use it** until I do one more PR *after* the merge.  The __print_function__ statement will make `print` without `()` a syntax error.

Commit:
Update aboki

Commit:
Update aboki

cclauss commented:
Please merge but do not use...  It contains syntax errors.

akinjide commented:
I can't merge if it contains syntax errors, but I can review and update the errors if its fine by you?

akinjide commented:
Can you also squash this commits. Thanks.

cclauss commented:
Can you please turn on this option?  https://help.github.com/articles/configuring-commit-squashing-for-pull-requests/

cclauss commented:
Once that is done, you should be able to do a "squash merge".

akinjide commented:
Cool! Done.

cclauss commented:
So, are you be able to do a "squash merge"?

akinjide commented:
I see the "squash merge" option, but I need to work on your branch before merging
